packages/hash/design-system/src/chip.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Chip } from "./chip";
+
+const meta = {
+  title: "Design System/Chip",
+  component: Chip,
+  tags: ["docsPage"],
+  args: {
+    label: "Example Chip",
+  },
+} satisfies Meta<typeof Chip>;
```

Comment:
What's the value of satisfies here vs just declaring meta as this type?

Reply:
This style comes from the official boilerplate, so there may be reasons beyond this simple example. I'd just stick with what's suggested upstream.